docker-compose.yml

```diff
@@ -1,9 +1,39 @@
-version: '3.7'
+version: "3.7"
 
 services:
   db:
     image: rethinkdb:latest
     ports:
-      - '8080:8080'
-      - '29015:29015'
-      - '28015:28015'
+      - "8080:8080"
+      - "29015:29015"
+      - "28015:28015"
+    volumes:
+      - rethink-data:/data
+    networks:
+      - parabol-network
+  redis:
+    image: redis
+    ports:
+      - "6379:6379"
+    volumes:
+      - redis-data:/data
+    networks:
+      - parabol-network
+  app:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.dev
```

Comment:
this doesn't look right, is this an old version? i thought we made this exclusively for prod

Reply:
I didn't, I left it as dev but editable for prod. At this phase, I don't really mind which way we go for now. This'll need one more refactor before it's really widely usable I think. Don't want to lose the work tho.

I'll update it to prod to make the intent more clear, then let's merge as to not lose the work?